src/installer/tests/Assets/Projects/HelloWorld_中文/HelloWorld_中文.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Can we put just this .csproj in the existing HelloWorld directory so that it uses the sources there? I think it the output directory and file names should be set based on the project name.